src/scss/components/button/_index.scss

```diff
@@ -1,3 +1,14 @@
+@import "vars";
 @import "mixins";
 
-// TBD... button styles
```

Comment:
There was a subset of styles that were needed to support Shadow DOM styling, but we technically didn't need to convert Button styles at this time. However, by the time I realized this, the work had already been done, so I'm including it with this PR as well. 

**Note:** Button styling hasn't been fully converted to SCSS, though. LESS files still exist because of other references from Light DOM styles and documentation styles.

Reply:
Changes to this file should be handled in a different PR.